database/insertData/02_permission_names.js

```diff
@@ -138,6 +138,20 @@ exports.queries = [
       }
     }
   }`,
+  // applyJoinCompany
+  `mutation applyJoinCompanyPermission {
+    createPermissionName(
+      input: {
+        permissionName: { 
+          id: 8500
+          name: "applyJoinCompany", permissionPolicyId: 2000 }
+      }
+    ) {
+      permissionName {
+        name
+      }
+    }
+  }`,
```

Comment:
Maybe can be removed - if we only use the general apply for linking new user to this core template